R/util-SaveData.R

```diff
@@ -0,0 +1,47 @@
+SaveData <- function(lData, lInputConfig) {
```

Comment:
@lauramaxwell was this meant to be exported/roxygened?

Reply:
yeah, we should have a roxygen header added here- it's only really applicable if we are using data from an external/cloud source, but will definitely be useful once we get that all set up, so it should be exported.

Reply:
Do we want this done for the upcoming release? Applies to `LoadData` as well then I suppose

Reply:
Yes, let's file an issue and assign to @samussiah. Need to make sure this works with a standard `lConfig` object.